src/utils.js

```diff
@@ -0,0 +1,20 @@
+export async function fetchData(url, idMeal) {
```

Comment:
This should be a generic function. Make it only accept the url and add a wrapper function that accepts both url and idMeal, created the url from url and idMeal then calls fetchData.